pkg/serverless/logs/logs.go

```diff
@@ -265,6 +266,10 @@ func (c *LambdaLogsCollector) ServeHTTP(w http.ResponseWriter, r *http.Request)
 }
 
 func processLogMessages(c *LambdaLogsCollector, messages []logMessage) {
+	// sort messages by time (all from the same time zone) in ascending order.
+	sort.Slice(messages, func(i, j int) bool {
```

Comment:
Just double checking. We know that messages arriving out of order both happens and has caused problems? Sorting is an expensive operation, so we obviously don't want to do it unless we have to.

Similarly, do we have a sense of how many messages we can expect to normally receive at a time? If we can expect just one or two, then sorting seems like a fine idea. But if it can get into the dozens, then that's a different story.

Reply:
Yes. Sorting is expensive. Sorting can mitigate most of the potential mismatch between log's request ID and context stored one.

Reply:
I created a new Release Candidate to make sure that sorting is not that harmful at least for these testing cases.
We will need to follow up on it for Prod env.